### PR TITLE
Add note on `influx backup` in 2.0.0-rc.0

### LINKED ISSUE #1560: Update back up data
Bc backup/restore has been temporarily removed, we need to update the "back up data" doc:
https://docs.influxdata.com/influxdb/v2.0/backup-restore/backup/

To back up data in 2.0 RC, you must stop `influxd` and manually copy the data directory.

### PR DESCRIPTION
**Pull request by pierwill**
Closes #1560.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/backup-restore/backup.md [hunk 1]
```diff
@@ -19,9 +19,16 @@ InfluxDB copies all data and metadata to a set of files stored in a specified di
 on your local filesystem.
 
 {{% warn %}}
-If you set up InfluxDB using **v2.0.0-beta.1** or earlier, you cannot back up data.
-Root tokens created prior to **v2.0.0-beta.2** do not have the necessary permissions.
-To succesfully use the backup tool, set up a new InfluxDB instance using **v2.0.0-beta.2+**.
+#### InfluxDB 2.0rc0
+
+The `influx backup` command is not compatible with InfluxDB 2.0rc0.
+To backup data, manually copy the InfluxDB data directories:
```
Comment (russorat) [inline content/influxdb/v2.0/backup-restore/backup.md:25]: you have to stop influxdb before doing this.

File: content/influxdb/v2.0/backup-restore/backup.md [hunk 2]
```diff
@@ -19,9 +19,19 @@ InfluxDB copies all data and metadata to a set of files stored in a specified di
...
+#### InfluxDB 2.0rc0
+
+The `influx backup` command is not compatible with InfluxDB 2.0rc0.
+To backup data,
+
+1. Stop `influxd`.
+2. Manually copy the InfluxDB data directories:
+
+   ```
+   cp -r [TK] [TK]
```
Comment (pierwill) [inline content/influxdb/v2.0/backup-restore/backup.md:31]: Should this be `cp -r ~/.influxdbv2 ~/.influxdb2_bak`?

Comment (russorat) [reply content/influxdb/v2.0/backup-restore/backup.md:31]: by default, the data directory will be ~/.influxdbv2, but of course they can change that if they want.

Comment (kelseiv) [reply content/influxdb/v2.0/backup-restore/backup.md]: Thanks, guys! To be explicit, we could add (by default, located in ~/.influxdbv2)

**Review by kelseiv — APPROVED** [review]

**Pull request merged**